Sources/SourceKitD/SourceKitD.swift

```diff
@@ -139,6 +139,8 @@ extension SourceKitD {
     timeout: Duration,
     fileContents: String?
   ) async throws -> SKDResponseDictionary {
+    try Task.checkCancellation()
```

Comment:
Worth checking this in `withCancellableCheckedThrowingContinuation` instead?

Reply:
Oh, we already checked there. Guess we just needed to move the log statement into `withCancellableCheckedThrowingContinuation`.

Reply:
Ah, we do too. Thanks 🙇‍♂️